### DISCUSSION TITLE
Fedora packaging

### DISCUSSION THREAD

**Discussion by gombosg** [moneymanagerex/moneymanagerex#4686] [General]
Hey, FYI I started the work to package `moneymanagerex` in Fedora. It's a big work thanks to the many dependencies, most of which need to be separated out.

- `lunasvg` - new package needed
- `wxsqlite` - new package needed compiled against wxWidgets 3.1

Plus build script changes needed to support local packaged, I started [patching in a local branch](https://github.com/moneymanagerex/moneymanagerex/compare/master...gombosg:fedora_packaging) and will submit the changes in a PR.

I'm probably not going to separate abandoned dependencies like `ChartNew` to separate packages.

We'll see how easy it will be to get everything reviewed, if there are issues, I'll set up a [copr repo](https://copr.fedorainfracloud.org/) instead.

Cheers!

**Comment by gombosg** [comment]
@vomikan tagging you here in case you don't follow the discussions board. Ping me if there are breaking changes in build scripts as I'm patching them.

**Reply by vomikan** [reply]
I don't follow builds on Linux at all. @PMaff,  if I'm not mistaken, is an expert in this field.

**Reply by gombosg** [reply]
Yeah, thanks, it's all fine, just need to split out dependencies in the `3rd` folder into subpackages (at least most of them). 🙂 
The app builds and runs fine on Linux otherwise!
